src/main/kotlin/io/rsocket/cli/Main.kt

```diff
@@ -244,9 +243,7 @@ class Main : Runnable {
         .also { showResponse(it) }
       stream -> client.requestStream(inputPayload).take(requestN)
         .collect { showResponse(it) }
-      channel -> client.requestChannel(flowOf(inputPayload).onRequest {
-        TODO("send next n requests")
-      }).take(requestN)
+      channel -> client.requestChannel(flowOf(inputPayload)).take(requestN)
```

Comment:
If you use `take(n)` in that case it's better to do `buffer` first if `requestN` is rather small.
By default, without `buffer` it will anyway `buffer(64)`
So it's also possible to do `buffer(n).take(n)` to send only one `requestN` frame

Reply:
Before (or while I fix this), how do I observe this? What's the flag to see RSocket frames?

Reply:
No logging added yet. Not sure, that now exists some fully MPP logging library. Some of them doesn't support native, some of them support only Android and iOS.
So I will try to do something internally in RSocket-kotlin (and may be later extract to lib, is possible)
You can create an issue with logging info, that you expect.
And also, ktor has logging feature, but not sure, how much it logs for websockets